types/index.d.ts

```diff
@@ -0,0 +1,434 @@
+    setZoomMax(zoomMax: number): this;
+    setWeightMin(weightMin: number): this;
+    setWeightMax(weightMax: number): this;
+    setColorGradient(colorGradient: Heatmap.ColorStop[]): this;
+    setOpacity(opacity: number): this;
+    setResolution(resolutionPixels: number): this;
+    setIntensityBias(intensityBias: number): this;
+    setIntensityScale(intensityScale: number): this;
+    setIndoorMapWithFloorId(indoorMapId: IndoorMapId, indoorMapFloorId: IndoorMapFloorId): this;
+    setElevation(elevation: number): this;
+    setElevationMode(mode: ElevationMode): this;
+    setPolygonPoints(polygonPoints: L.LatLngTuple): this;
+    setUseApproximation(useApproximation: boolean): this;
+}
+
+/* Wrld.indoors */
+
+namespace indoors {
+    
+    // eslint-disable-next-line no-unused-vars
+    class IndoorMap {
+        exit(): void;
+        getIndoorMapId(): IndoorMapId;
+        getIndoorMapName(): string;
+        getFloorCount(): number;
+        getFloors(): IndoorMapFloor;
+        getSearchTags(): { name: string; search_tag: string; icon_key: string}[];
+    }
+
+    class IndoorMapFloor {
+        // retain compat with existing API -- id was exposed as short name
+        // whereas it should really be the floorId (a.k.a. z_order)
+        getFloorId(): string;
+        getFloorIndex(): number;
+        getFloorName(): string;
+        getFloorShortName(): string;
+    }
+    
+    // eslint-disable-next-line no-unused-vars
+    class IndoorMapEntrance {
+        getIndoorMapId(): IndoorMapId;
+        getIndoorMapName(): string;
+        getLatLng(): L.LatLng;
+    }
+}
+
+/* Wrld.themes */
+
+namespace themes {
+
+    // eslint-disable-next-line no-unused-vars
+    const season = {
+        Spring: "Spring",
+        Summer: "Summer",
+        Autumn: "Autumn",
+        Winter: "Winter"
+    };
+
+    // eslint-disable-next-line no-unused-vars
+    const time = {
+        Dawn: "Dawn",
+        Day: "Day",
+        Dusk: "Dusk",
+        Night: "Night"
+    };
+
+    // eslint-disable-next-line no-unused-vars
+    const weather = {
+        Clear: "Default",
+        Overcast: "Overcast",
+        Foggy: "Foggy",
+        Rainy: "Rainy",
+        Snowy: "Snowy"
+    };
+
+}
+
+/* Wrld.buildings - TODO */
+
+namespace buildings {}
+
+/* Wrld.indoorMapEntities - TODO */
+
+namespace indoorMapEntities {}
+
+/* Wrld.spaindoorMapFloorOutlinesce - TODO */
+
+namespace indoorMapFloorOutlines {}
+
+/* Wrld */
+
+function map(element: HTMLElement | string, apiKey: string, options?: MapOptions): Map;
+function marker(latLng: L.LatLngExpression, options?: MarkerOptions): Marker;
+function popup(options?: PopupOptions, source?: L.Layer): Popup;
+function polygon(latlngs: L.LatLngTuple[] | L.LatLngTuple[][], options?: PolygonOptions): Polygon;
+function polyline(latlngs: L.LatLngExpression[], options?: PolylineOptions): Polyline;
+function prop(name: string, geometryId: string, location: L.LatLngExpression, options?: PropOptions): Prop;
```

Comment:
Same comment re: LatLngExpression here

Reply:
Shouldn't be an issue as mentioned above